x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/agent_list_page/index.tsx

```diff
@@ -438,6 +598,44 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
             </EuiFlexItem>
             <EuiFlexItem grow={1}>
               <EuiFilterGroup>
+                <EuiPopover
+                  ownFocus
+                  button={
+                    <EuiFilterButton
+                      iconType="arrowDown"
+                      onClick={() => setIsStatutsFilterOpen(!isStatusFilterOpen)}
+                      isSelected={isStatusFilterOpen}
+                      hasActiveFilters={selectedStatus.length > 0}
+                      numActiveFilters={selectedStatus.length}
+                      disabled={isAgentConfigsLoading}
+                    >
+                      <FormattedMessage
+                        id="xpack.ingestManager.agentList.statusFilterText"
+                        defaultMessage="Status"
+                      />
+                    </EuiFilterButton>
+                  }
+                  isOpen={isStatusFilterOpen}
+                  closePopover={() => setIsStatutsFilterOpen(false)}
+                  panelPaddingSize="none"
+                >
+                  <div className="euiFilterSelect__items">
+                    {statusFilters.map(({ label, status }, idx) => (
+                      <EuiFilterSelectItem
```

Comment:
forgot to add `key={}`?
Also - I don't see `idx` param being used (or is for the `key` prop?)

Reply:
it was for the key prop, I forgot to use it